apis/python/test/test_distance_metrics.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+import numpy as np
+import pytest
+from array_paths import *
+from sklearn.neighbors import NearestNeighbors
+
+from tiledb.vector_search import _tiledbvspy as vspy
+from tiledb.vector_search.ingestion import ingest
+from tiledb.vector_search.utils import load_fvecs
+
+siftsmall_uri = siftsmall_inputs_file
+queries_uri = siftsmall_query_file
+
+
+def normalize_vectors(vectors):
+    norms = np.linalg.norm(vectors, axis=1, keepdims=True)
+    return vectors / norms
+
+
+def test_cosine_similarity(tmp_path):
+    index_uri = os.path.join(tmp_path, "sift10k_flat_FLAT")
+    index = ingest(
+        index_type="FLAT",
+        index_uri=index_uri,
+        source_uri=siftsmall_uri,
+        source_type="FVEC",
+        distance_metric=vspy.DistanceMetric.COSINE,
+    )
+
+    dataset_vectors = load_fvecs(siftsmall_uri)
+    query_vectors = load_fvecs(queries_uri)
+
+    nn_cosine_sklearn = NearestNeighbors(n_neighbors=5, metric="cosine")
+    nn_cosine_sklearn.fit(dataset_vectors)
+    distances_sklearn, indices_sklearn = nn_cosine_sklearn.kneighbors(query_vectors)
+
+    distances, indices = index.query(query_vectors, k=5)
+
+    assert np.allclose(
+        distances_sklearn, distances, 1e-4
+    ), "Cosine similarity distances do not match"
+    assert np.array_equal(
+        indices_sklearn, indices
+    ), "Cosine similarity indices do not match"
+
+
+def test_inner_product(tmp_path):
```

Comment:
(Minor) We call these `test_cosine_similarity`, `test_inner_product`, and `test_l2_distance`. Shouldn't it be `test_cosine_distance` (b/c we're doing `1 - cosine similarity`) and `test_inner_product_distance`?